controllers/profiles/common/object_creators.go

```diff
@@ -147,7 +147,6 @@ func defaultContainer(workflow *operatorapi.SonataFlow) (*corev1.Container, erro
 			FailureThreshold:    healthStartedFailureThreshold,
 			PeriodSeconds:       healthStartedPeriodSeconds,
 		},
-		ImagePullPolicy: corev1.PullAlways,
```

Comment:
When we are deploying workflows for managed builds, we don't use the "latest" tag, so this is good.
But when we deploy workflows with images that has been built outside SonataFlow we are not setting any ImagePullPolicy here. But, ¿should we analyze the image that was produced externally? or this produced at a later point?

Reply:
We are, the mutator that applies the image to the deployment and also verifies if an external image has the latest tag or not. So it depends on the user to set `latest` or an actual tag version to their image.

Reply:
thanks!